sc2repo/README.md

```diff
@@ -2,3 +2,106 @@
 
 CAOM repository service for the CADC data engineering sandbox. 
 
+## configuration
+See the [cadc-tomcat](https://github.com/opencadc/docker-base/tree/master/cadc-tomcat) image
+docs for expected deployment and common config requirements. The `sc2repo` war file can be renamed
+at deployment time in order to support an alternate service name, including introducing
+additional path elements (see war-rename.conf).
+
+This service instance is expected to have a database backend to store the TAP metadata and which
+also includes the caom2 tables.
+
+Runtime configuration must be made available via the /config directory.
+
+### catalina.properties (cadc-tomcat)
+When running sc2repo.war in tomcat, parameters of the connection pool in META-INF/context.xml need to be configured in catalina.properties:
+
+```
+# database connection pools
+sc2repo.test.maxActive={max connections for test admin pool}
+sc2repo.test.username={username for test admin pool}
+sc2repo.test.password={password for test admin pool}
+sc2repo.test.url=jdbc:postgresql://{server}/{database}
+
+sc2repo.sandbox.maxActive={max connections for sandbox admin pool}
+sc2repo.sandbox.username={username for sandbox admin pool}
+sc2repo.sandbox.password={password for sandbox admin pool}
+sc2repo.sandbox.url=jdbc:postgresql://{server}/{database}
+```
+
+The `admin` account owns and manages (create, alter, drop) sandbox database objects and manages all the content (insert, update, delete).
+In addition, the TAP service does not currently support a configurable schema name: it assumes a schema 
+named `caom2` holds the content.
+
+### LocalAuthority.properties
```

Comment:
the example below is out of date with current requirements; 

please find and link to suitable documentation rather than create unmaintainable copies; if it doesn't exist in the right spot (in this case, cadc-registry) then create it there instead

Reply:
Moved the LocalAuthority.properties documentation to reg/cadc-registry README, and reference it in the sc2 README's.